content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
 
+### Collaborative writing and reviewing
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for more technical writing [@url:https://www.overleaf.com/ @url:https://www.authorea.com/].
+`TODO: survey other existing options, see if there are better references`
+These platforms typically offer version control and other features to support multi-author editing.
+`TODO: some use git in the back end, is that relevant?`
+However, their critical limitation for open writing is that they give all authors permission to modify all parts of the manuscript.
```

Comment:
Other limitations: lack of _issue_ support and on them it may be potentially hard to separate contributions. But ultimately we probably could have done this on another platform, so maybe "critical" is too strong of a word.